extensions/Khronos/KHR_lights/schema/examples/lights.gltf

```diff
@@ -0,0 +1,241 @@
+      "type": "VEC3"
+    },
+    {
+      "bufferView": 2,
+      "byteOffset": 0,
+      "componentType": 5125,
+      "count": 5172,
+      "max": [
+        3455
+      ],
+      "min": [
+        0
+      ],
+      "type": "SCALAR"
+    }
+  ],
+  "asset": {
+    "copyright": "2017 (c) Adobe Corp",
+    "minVersion": "2.0",
+    "version": "2.0"
+  },
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteLength": 41472,
+      "byteOffset": 0
+    },
+    {
+      "buffer": 1,
+      "byteLength": 41472,
+      "byteOffset": 0
+    },
+    {
+      "buffer": 2,
+      "byteLength": 20688,
+      "byteOffset": 0,
+      "target": 34963
+    }
+  ],
+  "buffers": [
+    {
+      "byteLength": 41472,
+      "uri": "MODEL_ROUNDED_CUBE_PART_1/positions.bin"
+    },
+    {
+      "byteLength": 41472,
+      "uri": "MODEL_ROUNDED_CUBE_PART_1/normals.bin"
+    },
+    {
+      "byteLength": 20688,
+      "uri": "MODEL_ROUNDED_CUBE_PART_1/indices.bin"
+    }
+  ],
+  "cameras": [
+    {
+      "name": "render_camera",
+      "perspective": {
+        "aspectRatio": 1.3333333730697632,
+        "yfov": 0.58904862403869629,
+        "zfar": 100,
+        "znear": 9.9999997473787516e-05
+      },
+      "type": "perspective"
+    },
+    {
+      "name": "render_camera2",
+      "perspective": {
+        "aspectRatio": 1.3333333730697632,
+        "yfov": 0.58904862403869629,
+        "zfar": 100,
+        "znear": 9.9999997473787516e-05
+      },
+      "type": "perspective"
+    }
+  ],
+  "materials": [
+    {
+      "doubleSided": true,
+      "name": "Rounded Cube Material",
+      "pbrMetallicRoughness": {
+        "baseColorFactor": [
+          0.63075679540634155,
+          0.63075679540634155,
+          0.63075679540634155,
+          1
+        ],
+        "metallicFactor": 0,
+        "roughnessFactor": 0.50300002098083496
+      }
+    }
+  ],
+  "meshes": [
+    {
+      "name": "MODEL_ROUNDED_CUBE_PART_1",
+      "primitives": [
+        {
+          "attributes": {
+            "NORMAL": 1,
+            "POSITION": 0
+          },
+          "indices": 2,
+          "material": 0,
+          "mode": 4
+        }
+      ]
+    }
+  ],
+  "nodes": [
+    {
+      "extensions": {
+        "KHR_lights": {
+          "light": 2
+        }
+      },
+      "matrix": [
+        1,
+        0,
+        0,
+        0,
+        0,
+        1,
+        0,
+        0,
+        0,
+        0,
+        1,
+        0,
+        0,
+        22,
+        0,
+        1
+      ],
+      "name": "point_light"
+    },
+    {
+      "name": "directional_light",
+      "extensions": {
+        "KHR_lights": {
+          "light": 1
+        }
+      }
+    },
+    {
+      "mesh": 0,
+      "name": "MODEL_ROUNDED_CUBE_PART_1model_N3D"
+    }
+  ],
+  "scene": 0,
+  "scenes": [
+    {
+      "name": "scene",
+      "nodes": [
+        0,
+        1,
+        2
+      ],
+      "extensions": {
+        "KHR_lights": {
+          "light": 0
```

Comment:
This suggests a scene may only contain a single ambient light; is that intended?

Reply:
Ah, just saw the comment above.